src/relation_engine_server/utils/arango_client.py

```diff
@@ -14,8 +14,10 @@
 
 def server_status():
     """Get the status of our connection and authorization to the ArangoDB server."""
+    auth = (_CONF['db_user'], _CONF['db_pass'])
+    adb_url = f"{_CONF['api_url']}/version"
     try:
-        resp = requests.get(_CONF['db_url'] + '/_api/database/current', auth=(_CONF['db_user'], _CONF['db_pass']))
+        resp = requests.get(adb_url, auth=auth)
```

Comment:
Rather than passing u/pwd every time, I think there's some way to get a token from ADB, right?

Reply:
That might be good, I haven't seen it yet.

Reply:
Although `python-arango` just does what you're doing so *shrug*